Source/EasyNetQ.Tests/PersistentChannelTests/When_a_channel_action_is_invoked.cs

```diff
@@ -8,7 +9,7 @@
 
 namespace EasyNetQ.Tests.PersistentChannelTests
 {
-    public class When_a_channel_action_is_invoked
+    public class When_a_channel_action_is_invoked: IDisposable
```

Comment:
```suggestion
    public class When_a_channel_action_is_invoked : IDisposable
```
and in other places

Reply:
It seems that you worked on code style rules. Could you fix them somehow to let IDE fix such kind of errors automatically or maybe some stuff is needed to run manually?

Reply:
This is really weird. I opened the solution and tried to change the code so that it violated the rules. Nothing happened. Something seems to have broken. I don't see syntax issues.